### DISCUSSION TITLE
vue-cesium编写思路

### DISCUSSION THREAD

**Discussion by test001-web** [zouyaoji/vue-cesium#231] [Q&A]
vue-cesium编写思路,有没有文档,或者图表,更容易理解源码

**Comment by zouyaoji** [comment]
没有哈。

就是在vue的生命周期里面把组件对应的cesium对象初始化了。vue-cesium自己维护了个创建cesium对象的生命周期。有个简单的调试方法。
你把这一行代码注释放开，就能看到相关组件的生命周期了。
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/private/use-log.ts#L55

**Reply by test001-web** [reply]
好的,感谢
